### PR TITLE
feat(pr-bot): impl pr bot

### PR DESCRIPTION
**Pull request by zhengjianhui**

### CODE REVIEW

File: pr-bot/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            for (const index in config.repos) {
+                const repo_info = new RepoInfo(config.repos[index].onwer, config.repos[index].repo);
+                const res = await repo_info.get_pr_info();
+                await discord.send_pr_infos(res);
+            }
+        } catch (e) {
+            console.log(e);
+        }
+
+    }, config.time_interval_for_notification * 60 * 1000);
```
Comment (KaoImin) [inline pr-bot/index.js:23]: Why 1000 here

Comment (zhengjianhui) [reply pr-bot/index.js:23]: > 

The hour is used here and the entry is in milliseconds, so the hour is converted to milliseconds here

Comment (KaoImin) [reply pr-bot/index.js:23]: Need another `* 60`

Comment (zhengjianhui) [reply pr-bot/index.js:23]: > Need another `* 60`

Oh! Right! , looks like I made a cheap mistake over here, I'll fix him right away

**Review by KaoImin — APPROVED** [review]

**Pull request merged**